services/ui-src/src/components/tables/EntityStatusIcon.tsx

```diff
@@ -0,0 +1,68 @@
+// components
+import { Image } from "@chakra-ui/react";
+// utils
+import { EntityShape } from "types";
+// assets
+import unfinishedIcon from "assets/icons/icon_error_circle_bright.png";
+import successIcon from "assets/icons/icon_check_circle.png";
+import { useContext, useMemo } from "react";
+import { ReportContext } from "components/reports/ReportProvider";
+import { mapValidationTypesToSchema } from "utils";
+import { object } from "yup";
+
+export const EntityStatusIcon = ({ entity }: Props) => {
+  const { report } = useContext(ReportContext);
+
+  const entityComplete = useMemo(() => {
+    const reportFormValidation = Object.fromEntries(
+      Object.entries(report?.formTemplate.validationJson ?? {}).filter(
+        ([key]) => key.includes("report_") || key.includes("state_")
+      )
+    );
+
+    const formValidationSchema =
+      mapValidationTypesToSchema(reportFormValidation);
+    const formResolverSchema = object(formValidationSchema || {});
+    mapValidationTypesToSchema;
```

Comment:
I don't think this line is doing anything?

Reply:
Yeah line 26 looks like it can be deleted.

Reply:
Indeed it does not!